templates/admin/library_report_detail.html

```diff
@@ -65,14 +65,8 @@ <h1 class="mx-auto">Boost {{ version.display_name }}</h1>
               <div>
                 <div class="grid grid-cols-5 gap-2">
                   {% for author in top_contributors_release_overall %}
-                    <div class="flex flex-col gap-y-2 w-20">
-                      {% if author.avatar_url %}
-                        <img src="{{ author.avatar_url }}"
-                             alt="{{ author.name }}"
-                             class="w-8 h-8 rounded mx-auto">
-                      {% else %}
-                        <div class="w-8 h-8 rounded bg-gray-300 mx-auto"></div>
-                      {% endif %}
+                    <div class="flex flex-col gap-y-2 w-20 items-center">
+                      {% avatar name=author.name image_url=author.avatar_url href=author.github_profile_url %}
```

Comment:
This is a django tag? It looks like like a partial.. what am I missing?

Reply:
It's similar to a partial.  
1. The way that it is created is slightly different.
```
{% include "partials/avatar.html" with name="" image_url="" href="" %}
```
2. Using a django tag ensures that the required fields are provided when using it.  If we use the `include` method instead, and don't include a name for example, it will still render but without a name.


I opted for the django tag as it's slightly easier to use but I'm fine with either way.